addons/product/pricelist_view.xml

```diff
@@ -2,11 +2,6 @@
 <openerp>
     <data>
 
-        <menuitem id="base.menu_sale_config_sales" name="Configuration"
```

Comment:
To check: removed access to pricelists in product app ? Shouldn't we keep a pricelist menu directly in the product menus ?